app.js

```diff
@@ -13,6 +15,67 @@ app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
 
+
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  // Retrieve an access token
+  spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
+
+
 // Our routes go here:
+app.get('/', (req, res) => {
+    // spotifyApi.getArtistAlbums('43ZHCT0cAZBISjO8DG9PnE').then(
+    //     function(data) {
+    //       console.log('Artist albums', data.body);
+    //     },
+    //     function(err) {
+    //       console.error(err);
+    //     }
+    //   );
+    res.render('home')
+});
+
+app.get('/artist-search', (req, res) => {
+    // console.log(data.body.artists.item)
+    spotifyApi
+    .searchArtists(req.query.title)
+    .then(data => {
+    console.log('Search artists', data.body.artists.items);
+    res.render('artistSearch', data.body)
```

Comment:
Here instead of just send the data on this way, it is a better approach to send as an object and give to the data a proprietary name
```res.render("albums", { artists: data.body.items });```

because, imagine that you want to send more than one data, so it will be better that each data has a name.